codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -64,6 +64,7 @@ public DefaultHttp2RemoteFlowController(Http2Connection connection,
                                             StreamByteDistributor streamByteDistributor) {
         this.connection = checkNotNull(connection, "connection");
         this.streamByteDistributor = checkNotNull(streamByteDistributor, "streamWriteDistributor");
+        streamByteDistributor.writer(writer);
```

Comment:
We probably don't need the `writer` member anymore ... could just be created anonymously here.